### ISSUE TITLE
Контроль выбора действия, как в комнате "Указатель"

### ISSUE THREAD

**Issue by alexyr** [yegorf1/RogueBot#282]
Если ввод не совпадает с предложенными вариантами, не засчитывать ход а вывести 'Такого выбора тебе не давали' и нужную клавиатуру.

Сейчас засчитывается последний вариант выбора, так как он попадает в else.

Это необходимо так как не всегда приходит сообщение с реакцией бота, и клавиатура не обновляется. В бою это может привести к смерти игрока.

![choice](https://cloud.githubusercontent.com/assets/1553411/21806999/3ddf41c4-d745-11e6-9975-527b640c9513.png)

**Comment by YogurtTheHorse** [comment]
@alexyr Не понял о чем ты вообще.

**Comment by thekiba** [comment]
@yegorf1 Я так понимаю, что варианты ответа не всегда приходят от бота. И если ввести любое сообщение, то засчитывается ход.

**Comment by YogurtTheHorse** [comment]
@thekiba Так, то есть баг в том, что не приходят сообщения, я правильно понял?

**Comment by thekiba** [comment]
@yegorf1 Агась

**Comment by alexyr** [comment]
@thekiba да, это.
@yegorf1 баг есть, но контроль ввода пользователя нужен! во всех 'else' нужно прописать повторный запрос ввода, а не последний вариант выбора

**Comment by YogurtTheHorse** [comment]
@alexyr А, вот в чем дело. Принято.
@thekiba, можешь лейблы нацепить?
